src/backend/base/langflow/graph/graph/base.py

```diff
@@ -335,9 +335,19 @@ async def _run(
             logger.exception(exc)
 
         try:
-            start_component_id = next(
-                (vertex_id for vertex_id in self._is_input_vertices if "chat" in vertex_id.lower()), None
+            # Prioritize the webhook component if it exists
+            start_component_id = None
+            webhook_component_id = next(
```

Comment:
would be nice here to have a List constact to enumerate all the "inputs" components, so when adding a new one it's much easier to spot this place in the code

Reply:
Absolutely